### ISSUE TITLE
More advanced relief handling

### ISSUE THREAD

**Issue by mbredif** [itownsResearch/demo-geoid#5]
For now, the 3D position is computed as :

`position = ellipsoid + ZFactor * geoid`

This feature request is for a slightly more complex definition where `relief` is another "elevation" image layer.

`position = ellipsoid + geoidFactor * geoid + reliefFactor * relief`

**Comment by LAREG-OJ** [comment]
Indeed, the full resquest would be to also be able to enhance the Earth flattening : 

`position = ellipsoid(flattening*flatteningFactor) + geoidFactor * geoid + reliefFactor * relief`

with a default fltattening (flatteningFactor=1) set to the real flattening of the Earth.
